source/includes/steps-install-bi-connector-rhel.yaml

```diff
@@ -6,6 +6,78 @@ source:
   ref: download-bi
 ---
 stepnum: 2
+title: Verify the integrity of the downloaded package.
+ref: verify-bi-rhel-package
+level: 4
+content: |
+  The MongoDB release team digitally signs all software packages to
+  certify that a particular MongoDB package is a valid and unaltered
+  MongoDB release. The `bi-connector.asc` key to validate the BI Connector is available on [pgp.mongodb.com](https://pgp.mongodb.com/bi-connector.asc)  
```

Comment:
The link look odd on the staging page.
<img width="775" alt="image" src="https://github.com/mongodb/docs-bi-connector/assets/91975317/584c01c9-06f6-4484-81bc-4e0d3b550ac2">